data/35e/wizards_of_the_coast/core/players_handbook/ph_equip.lst

```diff
@@ -131,7 +131,7 @@ Sword, Short			KEY:Sword (Short)				PROFICIENCY:WEAPON|Sword (Short)			TYPE:Weap
 Urgrosh, Dwarven			KEY:Urgrosh (Dwarven)			PROFICIENCY:WEAPON|Urgrosh (Dwarven)		TYPE:Weapon.Container.Melee.Exotic.Double.Standard.Dwarven.Slashing.Axe						ALTTYPE:Weapon.Melee.Exotic.Piercing	CONTAINS:UNLIM|Weapon Crystal=1											COST:50	WT:12		SLOTS:2				CRITMULT:x3	CRITRANGE:1	ALTCRITMULT:x3	ALTCRITRANGE:1	DAMAGE:1d8	ALTDAMAGE:1d6	EQMOD:STEEL			ALTEQMOD:STEEL	WIELD:TwoHanded												SIZE:M	SOURCEPAGE:p.121																																																			SPROP:A dwarven urgrosh is a double weapon. You can fight with it as if fighting with two weapons, but if you do, you incur all the normal attack penalties associated with fighting with fighting with two weapons, just as if you were using a one-handed weapon and a light weapon. The urgrosh's axe head is a slashing weapon that deals 1d8 points of damage. Its spear head is a piercing weapon that deals 1d6 damage. You can use either head as the primary weapon. The other is the off-hand weapon. A creature wielding a dwarven urgrosh in one hand can't use it as a double weapon - only one end of the weapon can be used in any given round. If you use a ready action to set an urgrosh against a charge, you deal double damage if you score a hit against a charging character. If you use an urgrosh against a charging character, the spear head is the part of the weapon that deals damage. An urgrosh is also called a spear-axe. Dwarves treat dwarven urgroshes as martial weapons.
-Waraxe, Dwarven			KEY:Waraxe (Dwarven)			PROFICIENCY:WEAPON|Waraxe (Dwarven)			TYPE:Weapon.Container.Melee.Martial.Exotic.Standard.Dwarven.Slashing.Axe												CONTAINS:UNLIM|Weapon Crystal=1											COST:30	WT:8							CRITMULT:x3	CRITRANGE:1							DAMAGE:1d10				EQMOD:STEEL						WIELD:TwoHanded												SIZE:M	SOURCEPAGE:p.121																							BONUS:WEAPON|WIELDCATEGORY|-1|PREVARGTEQ:DwarvenWaraxeExoticUse,1																		SPROP:A dwarf treats a dwarven waraxe as a martial weapon even when using it in one hand.
+Waraxe, Dwarven			KEY:Waraxe (Dwarven)			PROFICIENCY:WEAPON|Waraxe (Dwarven)			TYPE:Weapon.Container.Melee.Martial.Exotic.Standard.Dwarven.Slashing.Axe												CONTAINS:UNLIM|Weapon Crystal=1											COST:30	WT:8							CRITMULT:x3	CRITRANGE:1							DAMAGE:1d10				EQMOD:STEEL						WIELD:OneHanded	SIZE:M	SOURCEPAGE:p.121																							BONUS:WEAPON|WIELDCATEGORY|-1|PREVARGTEQ:DwarvenWaraxeExoticUse,1																		SPROP:A dwarf treats a dwarven waraxe as a martial weapon even when using it in one hand.
```

Comment:
Why are we changing this weapon?

Reply:
a dwarf can use it one handed, as per the SPROP, so this allows it to be equipped appropriately for a dwarf (a non-dwarf is unlikely to equip it really, though possibly a PRERACE could work?)

Reply:
The weapon already shifts from TwoHanded to OneHanded if the character is a dwarf. Much like the Bastard Sword shifts to Onehanded if the character has Exotic Weapon Bastard Sword feat. Unless the mechanic is not working, we should not just change one aspect of the item. As written, the weapon will become a LIGHT weapon for a Dwarf. 

That is why this tag exists: BONUS:WEAPON|WIELDCATEGORY|-1|PREVARGTEQ:DwarvenWaraxeExoticUse,1

Reply:
now that's perplexing, so it works now (just reverted the change and retested), previously I'd been unable to equip it two handed for a dwarf (to get the 1.5 strength mod) without this change.
Apologies for this, revering this one now.